frontend/src/components/pages/admin/AssessmentEditorPage.tsx

```diff
@@ -128,9 +133,7 @@ const AssessmentEditorPage = (): React.ReactElement => {
         test,
       },
     });
-
-    setCompletedForm(true);
-    history.push(ASSESSMENTS_PAGE);
+    resetDefaultValues();
```

Comment:
From the commit message:
> Now that we stay on the editor page sometimes after submission, we need to fix the handling of the reload prompt (otherwise, it will never display after the form is submitted for the first time). To do this, we check if the form state has been touched (dirtied), and only allow reloading if it has not. We also have to reset the form default values as this is how `rhf` determines whether the form is dirty.

This is the "reset the form default values" part at the end!